69-nodebase/.env.template

```
@@ -1,3 +1,7 @@
 
 # Database https://neon.com/
 DATABASE_URL="your_pg_neon_db_connection"
+
+# Btter Auth https://www.better-auth.com/
+BETTER_AUTH_SECRET="your_better_auth_secret"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)